agr_literature_service/api/crud/reference_crud.py

```diff
@@ -108,6 +110,8 @@ def create(db: Session, reference: ReferenceSchemaPost):  # noqa
                         obj_data["orcid_cross_reference"] = cross_reference_obj
                     del obj_data["orcid"]
                     db_obj = create_obj(db, AuthorModel, obj_data, non_fatal=True)
+                    if db_obj.name:
+                        authorNames += db_obj.name + '; '
```

Comment:
Are these authors at this point ordered by the 'order' field ?   I see that happens in get_citation_from_obj but not in get_citation_from_args, which uses authorNames from here ?

Reply:
Oh, this is only when creating a reference, so the authors are already ordered.